File: application/common/model/Room.php
<?php
namespace app\common\model;
use think\Model;

class Room extends Model {

    public function getId() {
        return (int)$this->data['id'];
    }

    public function getName() {
        return $this->data['name'];
    }

    public function getCapacity() {
        return $this->data['capacity'];
    }

    
}

File: api/src/main/java/com/example/api/entity/Room.java
package com.example.api.entity;

import io.swagger.annotations.ApiModelProperty;

import javax.persistence.*;

@Entity
public class Room {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @ApiModelProperty("教室ID")
    private Long id;

    @ApiModelProperty("教室名称")
    @Column(nullable = false)
    private String name = "";

    @ApiModelProperty("教室容量")
    private Long capacity;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getCapacity() {
        return capacity;
    }

    public void setCapacity(Long capacity) {
        this.capacity = capacity;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
}
